### DISCUSSION TITLE
Import Tags

### DISCUSSION THREAD

**Discussion by poudenes** [firefly-iii/firefly-iii#7426] [Firefly III]
Hi All,

Is it safe to remove all the import tags after a while?

I have a lot from 2022 and now already 2023. Every day 2 for 2 banks.

I now remove the "When selected Firefly III will add a tag to each imported transaction denoting the import; this groups your import under a tag." from the automated import. SO there will be no new tags created

Greeting,

**Comment by JC5** [comment]
That's safe to do yes 👍

**Comment by poudenes** [comment]
to understand: Because its only bundle the transaction that have that tag? Nothing more?

**Reply by JC5** [reply]
Exactly :)

**Comment by poudenes** [comment]
Thanks for the fast reply. Will close the discussion
